### PR TITLE
Add new alert and helpers

### PR DESCRIPTION
**Pull request by rfriberg**
- New "upsell" alert  (used on dashboard)
- Helpers for two features we keep reusing

Preview the alert here: https://precog.mapzen.com/mapzen/styleguide/rfriberg/new-alert/design-elements.html#alerts

### CODE REVIEW

**Review by souperneon — APPROVED** [review]

File: src/stylesheets/common/_helpers.scss [hunk 1]
```diff
@@ -172,14 +172,13 @@
   color: #333333;
 }
 
-.text-16 {
-  // font-weight: normal;
-  // font-size: 16px;
-  // line-height: 1.6;
+.deemphasize-text {
```
Comment (louh) [inline src/stylesheets/common/_helpers.scss]: I've seen this stock Bootstrap class used, that sounds like it's used in a very similar way: https://github.com/mapzen/styleguide/blob/77b0d5cb85ad8f08d56ff1a840182a415902d7eb/src/stylesheets/bootstrap/_type.scss#L101-L104

There is an example of `.text-muted` in use on the Settings page, under the Payment method heading where it says "(We won't charge you without your permission)". The color is #777777, which is the Bootstrap default for this class.

Should we rely on `.text-muted` instead (and adjust the text color to match our desired de-emphasized color)? Or do we have two levels of de-emphasized text (in which case we might want to use the same naming conventions, and document the difference between the two, with examples of situational uses)?

Comment (rfriberg) [reply src/stylesheets/common/_helpers.scss]: Oh right.  I'm even the one who used `text-muted` on that page.  We probably need to keep the two separate colors (#777 and #ccc are pretty far apart), but I can change this class name to match.  Good call.

Comment (rfriberg) [reply src/stylesheets/common/_helpers.scss]: And yeah, a section in the styleguide on different text treatments (and maybe background colors, too) would be a good addition.

**Comment by rfriberg** [comment]
Added sections for text helpers and background colors to the styleguide docs.

**Pull request merged**